apps/wallet/src/ui/app/pages/home/tokens/TokenStakingOverview.tsx

```diff
@@ -55,7 +49,7 @@ export function TokenStakingOverview({
     const isLoading = isPending || queryResultStake.isPending;
 
     return (
-        <Card type={CardType.Filled} onClick={handleOnClick} isDisabled={disabled}>
+        <Card type={CardType.Filled} onClick={handleOnClick} isDisabled={false}>
```

Comment:
I wouldnt remove the `disabled` prop of this component in case we need it in the future, just dont use it

Reply:
I mean the prop of `TokenStakingOverview`